boms/integration-tests/src/test/java/com/google/cloud/BomContentTest.java

```diff
@@ -98,6 +98,7 @@ private static void assertUniqueClasses(List<Artifact> allArtifacts)
 
       if (!currentArtifact.getGroupId().contains("google")
           || currentArtifact.getGroupId().contains("com.google.android")
+          || currentArtifact.getGroupId().contains("com.google.cloud.bigtable")
```

Comment:
@kolea2 I saw the duplicate classes exist between com.google.cloud.bigtable:bigtable-hbase-1.x-shaded:1.25.2-sp.2 and com.google.cloud.bigtable:bigtable-metrics-api:1.25.1. Because one of them is a shaded artifact, I'm excluding the artifacts from the class uniqueness check.
https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/2257#issuecomment-1004957628

If you think these duplicates are not supposed to happen, let me know.